cns/client_test.go

```diff
@@ -856,6 +856,28 @@ func TestClient(t *testing.T) {
 	}
 	t.Logf("Volume detached sucessfully")
 
+	// Test QueryVolumeAsync API only for vSphere version 7.0.3 onwards
```

Comment:
Just curious why moving up the test case of QueryVolumeAsync here?

Reply:
Moving it here will test the API when volume still exists. Previously it was at the bottom and invoked after deleteVolume. So queryAsync was returning empty volume list